ui.apps/src/main/webpack.core/internals/webpack.config.js

```diff
@@ -70,7 +70,14 @@ const WEBPACK_DEFAULT = {
               }
 
               // Load Autoprefixer AFTER Stylelint to avoid failing on Stylelint's prefix rules
-              plugins.push(require('autoprefixer'));
+              plugins.push(require('autoprefixer')({
```

Comment:
https://github.com/postcss/autoprefixer#browsers

autoprefixer's readme suggests a different route to include browserlist